app/presenters/finder_presenter.rb

```diff
@@ -345,4 +334,15 @@ def alert_query_string
   def eu_exit_finder
     slug == "/find-eu-exit-guidance-business"
   end
+
+  def facet_hashes(content_item_hash)
+    FacetExtractor.for(content_item_hash).extract
```

Comment:
What is a `FacetExtractor`? Forgive my ignorance, it looks this was already in the codebase, but I'm not sure what it is.